mysql/db/03_financial_records.sql

```diff
@@ -0,0 +1,11 @@
+USE finansu_db;
+
+CREATE TABLE financial_records (
+    id INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    year_id INT(10) UNSIGNED NOT NULL,
+    created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    PRIMARY KEY (id)
```

Comment:
```suggestion
    PRIMARY KEY (id),
```
カンマがないのでsynstax errorで実行されない🥺